src/settings/domain/models/Setting.ts

```diff
@@ -0,0 +1,8 @@
+export enum SettingName {
+  ZIP_DOWNLOAD_LIMIT = 'ZIP_DOWNLOAD_LIMIT'
```

Comment:
We should remove this SettingName enum once it is included in the js-dataverse package. The package will export it just like for the use cases. So we don't duplicate it in the SPA side.

Reply:
I always duplicate the types in the SPA, this way my imports look like
`import { Setting, SettingName } from '../settings/domain/models/Setting'`

instead of:
import { Setting,SettingName } from '@iqss/dataverse-client-javascript'`

Using the domain definitions instead of types from an external library so the code is not coupled to an external library. 

Just imagine that I need to add a new setting to the UI, if I were using js-dataverse type definition I'd would have to wait for this type to be updated in js-dataverse in order to use it in the SPA. By having my own SettingsName I can add new settings without waiting for js-dataverse to be updated, I mean, this is how I can develop new UI features without waiting for the backend/js-dataverse to be ready